docs/guides/security-policy-for-authors.md

```diff
@@ -346,24 +349,23 @@ on the CPANSec website.
 
 ## Response to Reports
 
-We aim to acknowledge your security report as soon as possible.
-However, this project is maintained by a single person in their
-spare time, and we cannot guarantee a rapid response.  If you have
-not heard back from the maintainers within a week, then please send
-a reminder to the maintainer and copy the report to CPANSec at
+The maintainer(s) aim to acknowledge your security report as soon as
+possible.  However, this project is maintained by a single person in
+their spare time, and they cannot guarantee a rapid response.  If you
+have not received a response from the them within a week, then
+please send a reminder to them and copy the report to CPANSec at
 <cpan-security@security.metacpan.org>.
```

Comment:
Would it be better to refer them to the reporting page?

https://security.metacpan.org/docs/report.html

Reply:
A link to that is just above this section.

As much as I want to keep the text brief and to the point, I think a note like "this project is maintained by a single person in
their spare time, and they cannot guarantee a rapid response" is important.